actors/miner/src/lib.rs

```diff
@@ -216,6 +225,69 @@ impl Actor {
         })
     }
 
+    /// Returns the owner address
+    fn get_owner(rt: &mut impl Runtime) -> Result<GetOwnerReturn, ActorError> {
+        rt.validate_immediate_caller_accept_any()?;
+        let state: State = rt.state()?;
+        let owner = get_miner_info(rt.store(), &state)?.owner;
+        Ok(GetOwnerReturn { owner })
+    }
+
+    /// Returns the worker address
+    fn get_worker(rt: &mut impl Runtime) -> Result<GetWorkerReturn, ActorError> {
+        rt.validate_immediate_caller_accept_any()?;
+        let state: State = rt.state()?;
+        let worker = get_miner_info(rt.store(), &state)?.worker;
+        Ok(GetWorkerReturn { worker })
+    }
+
+    /// Returns the control addresses
+    fn get_controls(rt: &mut impl Runtime) -> Result<GetControlsReturn, ActorError> {
+        rt.validate_immediate_caller_accept_any()?;
+        let state: State = rt.state()?;
+        let controls = get_miner_info(rt.store(), &state)?.control_addresses;
+        Ok(GetControlsReturn { controls })
+    }
+
+    /// Returns the miner's sector size
+    fn get_sector_size(rt: &mut impl Runtime) -> Result<GetSectorSizeReturn, ActorError> {
+        rt.validate_immediate_caller_accept_any()?;
+        let state: State = rt.state()?;
+        let sector_size = get_miner_info(rt.store(), &state)?.sector_size;
+        Ok(GetSectorSizeReturn { sector_size })
+    }
+
+    /// Returns the miner's total funds locked as pre_commit_deposit
+    fn get_pre_commit_deposit(
+        rt: &mut impl Runtime,
+    ) -> Result<GetPreCommitDepositReturn, ActorError> {
+        rt.validate_immediate_caller_accept_any()?;
+        let state: State = rt.state()?;
+        Ok(GetPreCommitDepositReturn { pre_commit_deposit: state.pre_commit_deposits })
+    }
+
+    /// Returns the sum of the initial pledge requirements of all active sectors of this miner.
+    fn get_initial_pledge(rt: &mut impl Runtime) -> Result<GetInitialPledgeReturn, ActorError> {
+        rt.validate_immediate_caller_accept_any()?;
+        let state: State = rt.state()?;
+        Ok(GetInitialPledgeReturn { initial_pledge: state.initial_pledge })
+    }
+
+    /// Returns the absolute value of debt this miner owes from unpaid fees.
+    fn get_fee_debt(rt: &mut impl Runtime) -> Result<GetFeeDebtReturn, ActorError> {
+        rt.validate_immediate_caller_accept_any()?;
+        let state: State = rt.state()?;
+        Ok(GetFeeDebtReturn { fee_debt: state.fee_debt })
+    }
+
+    /// Returns the absolute value of the locked funds in this miner's vesting table.
+    /// Note that this does NOT include other types of locked funds like precommit deposits.
+    fn get_locked_funds(rt: &mut impl Runtime) -> Result<GetLockedFundsReturn, ActorError> {
```

Comment:
Let's rename this to be more descriptive so the comment isn't needed. Maybe get_vesting_funds_total (and we can add get_vesting_funds_table later).